linkerd/tests/common.py

```diff
@@ -540,3 +549,26 @@
 EXPECTED_METRICS_V2_E2E = {
     k: aggregator.COUNT if v == aggregator.MONOTONIC_COUNT else v for k, v in EXPECTED_METRICS_V2.items()
 }
+
+EXPECTED_METRICS_V2_NEW = {}
+
+for metric_name, metric_type in list(EXPECTED_METRICS_V2.items()):
```

Comment:
Rather than hard code a new mapping, I thought it'd be useful to show the changes